### PR TITLE
feat(api): replace up arrow with period

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#673

Ref: 673

### Description

Replace up arrow with period

### Release Plan

- [ ] Merge once approved

### CODE REVIEW

**Review by leite08 — APPROVED** [review]

File: api/app/src/external/commonwell/document/document-query.ts [hunk 1]
```diff
@@ -208,8 +208,16 @@ async function downloadDocsAndUpsertFHIR({
...
-        if (doc.content?.masterIdentifier?.value && doc.content.location) {
-          const { writeStream, promise } = uploadStream(doc.content.masterIdentifier.value);
+        const primaryId = getDocumentPrimaryId(doc);
+
+        const s3FileName = createS3FileName(patient.cxId, primaryId);
+
+        const s3File = await fileExists(s3FileName);
+
+        if (s3File) return;
+
```
Comment (leite08) [inline api/app/src/external/commonwell/document/document-query.ts:218]: nit: lots of spaces 😄

**Review by leite08 — APPROVED** [review]
It looks good!

**Review by leite08 — APPROVED** [review]
It looks good!

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**